docs/functions.rst

```diff
@@ -1572,6 +1572,65 @@ Example:
         args:
           name: "test-snapshot-content-content-dfc8fa67-8b11-4fdf-bf94-928589c2eed8"
 
+BackupDataUsingKopiaServer
+--------------------------
+
+This function backs up data from a container into any object store
+supported by Kanister using Kopia Repository Server as Data-mover.
+
+.. note::
+   It is important that the application includes a ``kanister-tools``
+   sidecar container. This sidecar is necessary to run the
+   tools that capture path on a volume and store it on the object store.
```

Comment:
I might have missed, did we discuss about this already somewhere? Adding sidecar manually in an application doesn't seem to be a good user experience according to me.
If it has already been discussed we will have to improve this document to also explain how to add the sidecar.

Reply:
We are following a similar workflow as of the existing `BackupData` function (https://docs.kanister.io/functions.html#backupdata). We can track this as a separate issue.

Reply:
yeah makes sense, I understand. @r4rajat  Let's make sure that we raise another PR that explains how do we do that.